src/game.cpp

```diff
@@ -2165,16 +2165,24 @@ int game::inventory_item_menu( int pos, int iStartX, int iWidth,
 bool game::handle_mouseview( input_context &ctxt, std::string &action )
 {
     cata::optional<tripoint> liveview_pos;
+    auto &mgr = panel_manager::get_manager();
+    const bool sidebar_right = get_option<std::string>( "SIDEBAR_POSITION" ) == "right";
+    int width = sidebar_right ? mgr.get_width_right() : mgr.get_width_left();
     do {
         action = ctxt.handle_input();
         if( action == "MOUSE_MOVE" ) {
             const cata::optional<tripoint> mouse_pos = ctxt.get_coordinates( w_terrain );
             if( mouse_pos && ( !liveview_pos || *mouse_pos != *liveview_pos ) ) {
                 liveview_pos = mouse_pos;
                 liveview.show( *liveview_pos );
+                draw_panels( true );
+                catacurses::window &w = catacurses::newwin( TERMY / 2, width,
```

Comment:
```suggestion
                const catacurses::window &w = catacurses::newwin( TERMY / 2, width,
```
GCC complained about `invalid initialization of non-const reference of type 'catacurses::window&' from an rvalue of type 'catacurses::window'`

Reply:
Changed. Tested to make sure it still works the same.